pkg/stanza/operator/operator.go

```diff
@@ -38,8 +38,6 @@ type Operator interface {
 	CanProcess() bool
 	// ProcessBatch processes a batch of entries from an operator.
 	ProcessBatch(context.Context, []*entry.Entry) error
-	// Process will process an entry from an operator.
-	Process(context.Context, *entry.Entry) error
```

Comment:
This is a breaking change in public API of the package `github.com/open-telemetry/opentelemetry-collector-contrib/pkg/stanza/operator`. If we want to remove this, probably deprecating first would be good? What do you think @djaglowski?